### DISCUSSION TITLE
tutorials/programmability/ansible/intent-based-management/

### DISCUSSION THREAD

**Discussion by giscus** [srl-labs/learn-srlinux#113] [comments]
# tutorials/programmability/ansible/intent-based-management/

Open documentation for open Network OS

https://learn.srlinux.dev/tutorials/programmability/ansible/intent-based-management/

**Comment by Pegasust** [comment]
This is exciting. Since we're talking about intent-based management with Ansible, while it's the industry's go-to, is there any love to some type-safety onto configuration layer (using JSON/YAML generating functional languages)?

**Reply by hellt** [reply]
Hi @Pegasust
what do you mean exactly?
